libafl/src/stages/tmin.rs

```diff
@@ -393,7 +396,16 @@ where
         let obs = observers
             .get(self.observer_handle())
             .expect("Should have been provided valid observer name.");
-        Ok(obs.as_ref().hash_simple() == self.orig_hash)
+        let res = obs.as_ref().hash_simple() == self.orig_hash;
+        #[cfg(feature = "track_hit_feedbacks")]
+        {
+            self.last_result = Some(res);
+        }
+        Ok(res)
+    }
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn last_result(&self) -> Result<bool, Error> {
+        self.last_result.ok_or(Error::premature_last_result())
```

Comment:
Instead of `ok_or` we could directly store a `Result<...>` in `last_result`, which means the result is faster (but we lose the perfect mapping of where the error was triggered)

Reply:
Not useful due to Clone